index.html

```diff
@@ -97,5 +97,128 @@ <h1 class="primary-text">Hey There. I'm Alan</h1>
+                </p>
+                <ul class="tag-list white-font">
+                    <li class="card-feature-items">html</li>
+                    <li class="card-feature-items">bootstrap</li>
+                    <li class="card-feature-items">Ruby</li>
+                </ul>
+            </div>
+            <div class="button-section">
+                <a class="card-button white-font" href="#">See Project</a>
+            </div>
+        </div>
+        <div class="work-card">
+            <div class="info-holder">
+                <h2 class="project-heading white-font">Profesional Art Printing Data</h2>
+                <p class="card-description white-font">A daily selection of privately personalized reads;
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="tag-list white-font">
+                    <li class="card-feature-items">html</li>
+                    <li class="card-feature-items">bootstrap</li>
+                    <li class="card-feature-items">Ruby</li>
+                </ul>
+            </div>
+            <div class="button-section">
+                <a class="card-button white-font" href="#">See Project</a>
+            </div>
+        </div>
+        <div class="work-card">
+            <div class="info-holder">
+                <h2 class="project-heading white-font">Profesional Art Printing Data</h2>
+                <p class="card-description white-font">A daily selection of privately personalized reads;
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="tag-list white-font">
+                    <li class="card-feature-items">html</li>
+                    <li class="card-feature-items">bootstrap</li>
+                    <li class="card-feature-items">Ruby</li>
+                </ul>
+            </div>
+            <div class="button-section">
+                <a class="card-button white-font" href="#">See Project</a>
+            </div>
+        </div>
+        <div class="work-card">
+            <div class="info-holder">
+                <h2 class="project-heading white-font">Profesional Art Printing Data</h2>
+                <p class="card-description white-font">A daily selection of privately personalized reads;
+                    no accounts or sign-ups required. has been the industry's standard
+                </p>
+                <ul class="tag-list white-font">
+                    <li class="card-feature-items">html</li>
+                    <li class="card-feature-items">bootstrap</li>
+                    <li class="card-feature-items">Ruby</li>
+                </ul>
+            </div>
+            <div class="button-section">
+                <a class="card-button white-font" href="#">See Project</a>
+            </div>
+        </div>
+
+    </section>
```

Comment:
While you did a tremendous job adding the work section you forgot a very important part of this project adding the about section and the skills section. Please kindly add the About section and the skills section inorder for you project to be fully complete. 

Add this section: 
![Screenshot from 2022-05-28 15-15-01](https://user-images.githubusercontent.com/79506105/170823262-566c0ad2-2473-4efa-94f0-0d89e5b0b7ba.png)